profile/rwahs.xml

```diff
@@ -7924,6 +7947,113 @@
         </placement>
       </bundlePlacements>
     </display>
+    <display code="photograph_search_display" type="ca_objects" system="1">
+      <labels>
+        <label locale="en_AU">
+          <name>Photograph Object search</name>
+        </label>
+      </labels>
+      <typeRestrictions>
+        <restriction type="photograph"/>
+      </typeRestrictions>
+      <groupAccess>
+        <permission group="admin" access="edit"/>
+        <permission group="library" access="read"/>
+        <permission group="museum" access="read"/>
+        <permission group="photograph" access="read"/>
+        <permission group="memorial" access="read"/>
+      </groupAccess>
+      <bundlePlacements>
+        <placement code="ca_objects_preferred_labels">
+          <bundle>ca_objects.preferred_labels</bundle>
+          <settings>
+            <setting name="label" locale="en_AU">Title</setting>
+            <setting name="format"><![CDATA[<div style="min-width:30em;">^preferred_labels</div>]]></setting>
```

Comment:
This is where the title was too long making the table look funny (don't have time to find the issue number but it's in youtrack), and this is the fix that you put in for the display ;) Needs both